lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -112,7 +112,8 @@ def event_action_params(event)
         :_id => @document_id ? event.sprintf(@document_id) : nil,
         :_index => event.sprintf(@index),
         :_type => type,
-        :_routing => @routing ? event.sprintf(@routing) : nil
+        :_routing => @routing ? event.sprintf(@routing) : nil,
+        :parent => @parent ? event.sprintf(@parent) : nil
```

Comment:
This is breaking some old tests. Rather than setting this to nil we should simply not set the key if it isn't present.

We should probably go back and do the same for routing as well, but that's beyond the scope of this PR.